kahuna/public/stylesheets/main.css

```diff
@@ -1233,7 +1234,7 @@ FIXME: what to do with touch devices
    Labeller
    ========================================================================== */
 .labeller {
-    min-height: 20px;
+    height: 25px;
```

Comment:
Fix height because hover state with no labels has a mysterious dark grey bit when the image next to it has labels.

![screen shot 2015-07-17 at 13 52 39](https://cloud.githubusercontent.com/assets/836140/8747660/ba878ff2-2c8c-11e5-89e7-75242b103d2d.png)

![screen shot 2015-07-17 at 13 53 20](https://cloud.githubusercontent.com/assets/836140/8747670/d2a975c8-2c8c-11e5-9766-1b97d7759ca8.png)